shared-bindings/memorymonitor/AllocationSize.c

```diff
@@ -1,4 +1,4 @@
-/*
+>/*
```

Comment:
This looks like a typo. I don't think it'll break anything though because it is off by default.